src/client/components/military-service/ServiceInformationSection.jsx

```diff
@@ -12,15 +12,33 @@ import { veteranUpdateField } from '../../actions';
  * `reviewSection` - Boolean. Hides components that are only needed for ReviewAndSubmitSection.
  */
 class ServiceInformationSection extends React.Component {
+  constructor() {
+    super();
+    this.displayLabel = this.displayLabel.bind(this);
+  }
+
+  displayLabel(fullList, userValue) {
```

Comment:
I didn't originally do this because `calculated.js` seemed like it was supposed to be for things related to state but I added it with a comment.